packages/amplication-server/src/core/topic/topic.service.ts

```diff
@@ -39,6 +45,24 @@ export class TopicService extends BlockTypeService<
     return super.create(args, user);
   }
 
+  async createDefault(resource: Resource, user: User): Promise<Topic> {
+    const resourceNameKebabCase = kebabCase(resource.name.toLowerCase());
+    const resourceNameTopicConvention = resource.name
+      .toLowerCase()
+      .replace(/[\s\-/]/g, ".");
+
+    const defaultTopic: CreateTopicArgs = {
+      data: {
+        displayName: `${resourceNameKebabCase}-${DEFAULT_TOPIC_DISPLAY_NAME}`,
+        name: `${resourceNameTopicConvention}.${DEFAULT_TOPIC_NAME}`,
```

Comment:
I think we can go with a fixed name like topic.sample

Reply:
I left a comment earlier here, asking what happens when there is more than one message broker. That's what I did it like that

Reply:
> @abrl91 I added a small comment about the name of the default topic. One more thing - I expected to see in the video the default topic in the pending changes list, just like we see the User entity when creating a Service.

Yes. I know. Please read my comment 😎
https://github.com/amplication/amplication/pull/4675#issuecomment-1356323028

Reply:
The name should be unique per broker, and if it is not like that now, it sounds to me like a bug